website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md

```diff
@@ -0,0 +1,199 @@
+   ```
+
+2. To stop keycloak, run the following command:
+
+   ```bash
+   supervisorctl stop keycloak
+   ```
+
+3. To exit the remaining process, run the following command:
+
+   ``` bash
+   kill -9 `pgrep -f keycloak`
+   ```
+
+4. To start keycloak backup, run the following command
+
+   ``` bash
+   /bin/sh /opt/keycloak/bin/standalone.sh -c standalone.xml -b 0.0.0.0 -Djboss.socket.binding.port-offset=1 -Dkeycloak.migration.action=export -Dkeycloak.migration.provider=singleFile -Dkeycloak.migration.file=/tmp/keycloak_backup.json
+   ```
+
+5. Monitor the output and look for similar lines:
+
+   ```bash
+   07:25:11,805 INFO  [org.hibernate.validator.internal.util.Version] (ServerService Thread Pool -- 57) HV000001: Hibernate Validator 6.0.22.Final
+   07:25:12,930 INFO  [org.hibernate.hql.internal.QueryTranslatorFactoryInitiator] (ServerService Thread Pool -- 57) HHH000397: Using ASTQueryTranslatorFactory
+   07:25:13,691 INFO  [org.keycloak.services] (ServerService Thread Pool -- 57) KC-SERVICES0033: Full model export requested
+   07:25:13,691 INFO  [org.keycloak.exportimport.singlefile.SingleFileExportProvider] (ServerService Thread Pool -- 57) Exporting model into file /appsmith-stacks/data/keycloak_bkp/keycloak_backup.json
+   07:25:14,488 INFO  [org.keycloak.services] (ServerService Thread Pool -- 57) KC-SERVICES0035: Export finished successfully
+   ```
+
+   When you see lines similar to this, press <kbd>ctrl+c</kbd> to stop the process.
+
+6. To exit out of the shell, run following command:
+
+   ```bash
+   exit
+   ```
+
+7. To download the backup file available at `/tmp/keycloak_backup.json`, run the following command:
+
+   ```bash
+   kubectl cp <namespace>/appsmith-0:/tmp/keycloak_backup.json ./keycloak_bkp.json
+   ```
+
+## Uninstall Helm chart
+
+To uninstall the existing helm chart, run the following command:
+
+```bash
+helm uninstall appsmith -n <namespace>
+```
+
+## Configure parameters
+
+To ensure that the Business Edition Helm chart runs, you need to make some changes to the `values.yaml` file. Follow the steps below to configure parameters:
+
+1. Add the following snippet to the bottom of your `values.yaml` file. This snippet enables PostgreSQL.
+
+   ```yaml
+   postgresql:
+     enabled: true
+     auth:
+       username: root
+       password: <PASSWORD>
+       postgresPassword: <POSTGRESQL_PASSWORD>
+       database: keycloak
+   ```
+
+2. Choose, or create a shared file system. For more information, see [Create a shared file system.](/getting-started/setup/installation-guides/kubernetes/business-edition#create-a-shared-file-system)
+
+3. Add the license key and a few other variables related to keycloak to `applicationConfig` section:
+
+   ```yaml
+     APPSMITH_LICENSE_KEY: ""
+     APPSMITH_KEYCLOAK_DB_DRIVER: ""
+     APPSMITH_KEYCLOAK_DB_USERNAME: ""
+     APPSMITH_KEYCLOAK_DB_PASSWORD: ""
+     APPSMITH_KEYCLOAK_DB_URL: ""
+     APPSMITH_KEYCLOAK_DB_NAME: "keycloak"
+   ```
+
+## Install Helm chart 
```

Comment:
```suggestion
## Install new Helm chart 
```

Reply:
Let's discuss this.

Reply:
I saw the new change. Keep it as "Install Helm Chart v2"